SQF/dayz_code/stringtable.xml

```diff
@@ -11279,13 +11279,16 @@
 			<Czech>Musíte mít syrové maso</Czech>
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_19">
-			<English>Wrong code entered too many times, wait 10 seconds</English>
+			<English>You entered the wrong code, wait %1 seconds.</English>
 			<German>Es wurde zu oft der falsche Code eingetippt. Du musst nun 10 Sekunden warten</German>
 			<Russian>Неправильный код введён слишком много раз, подождите 10 секунд</Russian>
 			<Dutch>Je hebt te vaak een foute code ingevoerd. 10 seconden geduld svp.</Dutch>
 			<French>Trop d'erreurs avec le code, patientez 10 secondes</French>
 			<Czech>Mnohokrát jsi zadal špatný kód, musíš počkat 10 vteřin</Czech>
 		</Key>
+		<Key ID="STR_EPOCH_PLAYER_19_WAIT">
+			<English>You must wait another %1 seconds before you can enter another code.</English>
+		</Key>
```

Comment:
For the german string:

Du musst weitere %1 Sekunden warten, bevor du einen anderen Code eingeben kannst.